bigquery_etl/glean_usage/templates/baseline_clients_daily_v1.sql

```diff
@@ -130,11 +130,25 @@ windowed AS (
       ORDER BY
         submission_timestamp
     )
+),
+joined as (
+  SELECT
+    cd.* EXCEPT (_n),
+    cfs.first_seen_date,
+    -- the first seen date may be earlier than the submission date since it also
+    -- takes into account the migration ping
+    (cd.submission_date = cfs.first_seen_date) AS is_new_profile
+  FROM
+    windowed AS cd
+  LEFT JOIN
+    `{{ first_seen_table }}` AS cfs
+  USING
+    (client_id)
+  WHERE
+    _n = 1
 )
 --
-SELECT
-  * EXCEPT (_n)
-FROM
-  windowed
-WHERE
-  _n = 1
+select
```

Comment:
```suggestion
SELECT
```

Reply:
If you can run script/format_sql on the output of this, it should catch any remaining style inconsistencies.

Reply:
I've explicitly ignored these files in the format_sql script, because the the double braces of the jinja templating tends to break.